experiment/manual-trigger/manual-trigger.go

```diff
@@ -20,15 +20,17 @@ package main
 import (
 	"flag"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"net/url"
 	"strings"
 
+	"k8s.io/test-infra/prow/config"
 	"k8s.io/test-infra/prow/github"
 	"k8s.io/test-infra/prow/jenkins"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/pod-utils/downwardapi"
+
+	"github.com/sirupsen/logrus"
```

Comment:
nit: this import should be in it's own group after stdlib but before `k8s.io/test-infra`